snapcraft/main.py

```diff
@@ -19,82 +19,87 @@
 snapcraft
 
 Usage:
-  snapcraft [--version | --help] [options] COMMAND [ARGS ...]
   snapcraft [options]
+  snapcraft [options] init
+  snapcraft [options] pull [<part> ...]
+  snapcraft [options] build [<part> ...]
```

Comment:
Now that these are all together, should `--no-parallel-build` be an option specific to build?